src/linux_cpu_system_measurement.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "linux_cpu_system_measurement.hpp"
+
+#include <string>
+#include <vector>
+
+LinuxCPUSystemMeasurement::LinuxCPUSystemMeasurement()
+: numProcessors_(0)
+{
+  initCPUProcess();
+  makeReading();
+  std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
Sleeping in a constructor seems like a really bad idea.

If you move the sleep in `main()` to be the first thing in your `while(true)` and eliminate the sleep here, it should have the same behavior, right?

Reply:
this sleep is to have a first good cpu system reading